Assets/Scripts/Gamestate/AIManager.cs

```diff
@@ -80,7 +102,22 @@ private void OnDamageTaken(HealthController healthController, float damage, Dama
         {
             lastPlayerThatHitMe = info.sourcePlayer;
         }
+        if (info.damageType != DamageType.Explosion)
+            return;
+        StartCoroutine(WaitAndToggleAgent());
+    }
+
+    public IEnumerator WaitAndToggleAgent()
+    {
+        agent.enabled = false;
+        body.isKinematic = false;
+        colliderBox.isTrigger = false;
+        yield return new WaitForSeconds(0.5f);
```

Comment:
Huh, so this seems to work fine, but I'm surprised it doesn't appear to cause issues. I guess the bots are stunned after explosions and so on.

Reply:
Yep, they are stunned, but it works fine as the AIs are still travelling with the knockback at that point